source/manual/dns.html.md

```diff
@@ -15,76 +15,102 @@ GOV.UK is responsible for managing several DNS zones, spanning a number of `*.go
 
-- The domain name `gov.uk.` is an apex domain so it [cannot have a CNAME record](https://tools.ietf.org/html/rfc1912#section-2.4).
-  Instead, it has A records that point directly to anycast virtual IP addresses (VIPs) for our CDN provider.
-- `www.gov.uk.` is a CNAME to `www-cdn.production.govuk.service.gov.uk.`, which
-  means we do not need to make a request to Jisc if we want to change CDN
-  providers. We can just change where the CNAME points to.
+The `data.gov.uk` redirect to `www.data.gov.uk` is [configured in govuk-fastly-secrets](https://github.com/alphagov/govuk-fastly-secrets/blob/61e0206f62a7af6e45c80820e90d52db7590f3ab/secrets.yaml#L387-L396).
 
-## DNS for non-`gov.uk` domains
+There are a number of other subdomains of `data.gov.uk` configured in govuk-dns-tf.
 
-GOV.UK also manages DNS zones for some non-`gov.uk` domains (e.g. `independent-inquiry.uk`).
+### The `blog.gov.uk` domain
 
-These should be managed in Terraform, with each domain having its own zone configuration file in [govuk-dns-tf][].
+This is managed in GOV.UK's Jisc account.
 
-[govuk-dns-tf]: https://github.com/alphagov/govuk-dns-tf
+We have a [wildcard CNAME](https://github.com/alphagov/govuk-dns-tf/blob/8fa490bce3d9272e6df69f4dbcb1c1be1b3f07c2/zones/blog.gov.uk.yaml#L18-L21) delegating all subdomains of `blog.gov.uk` to our provider, DXW.
+
+### The `campaign.gov.uk` domain
+
+This is managed in GOV.UK's Jisc account.
+
+We have a [wildcard CNAME](https://github.com/alphagov/govuk-dns-tf/blob/1be5ae58e82fb47f0e42cc6f7c2507b424fa9200/zones/campaign.gov.uk.yaml#L91-L95) delegating all subdomains of `campaign.gov.uk` to our provider, DXW.
+
+There are a handful of campaign subdomains that have their own specific NS records or CNAME, which take precedence over the wildcard ([example](https://github.com/alphagov/govuk-dns-tf/blob/1be5ae58e82fb47f0e42cc6f7c2507b424fa9200/zones/campaign.gov.uk.yaml#L54-L60)).
+
+### The `independent.gov.uk` domain
+
+The domain itself is owned by Cabinet Office, and its delegation is also managed by Cabinet Office.
+
+[The management of the domain is being looked at](https://trello.com/c/qNpyVaC5/3228-consolidate-co-vs-non-co-domains-in-govuks-jisc-account) by Platform Security & Reliability.
+
+Relatedly, there are a [number of `independent-*.uk` domains managed by GOV.UK](#other-domains-we-manage).
+
+### The `api.gov.uk` domain
+
+The domain itself is owned by Cabinet Office, but delegated to GOV.UK, who then delegate it back to Cabinet Office.
```

Comment:
wat

Reply:
Yes, exactly! CO/CDDO already manage the third level `api.gov.uk` domain in Jisc/Nominet but currently point to GDS to define the name servers for the `*.api.gov.uk` fourth level domains. And we then just point `www` to a CDDO-owned CNAME (and also manage a handful of other api subdomains).

They own the guidance at https://www.gov.uk/guidance/get-an-api-domain-on-govuk and they manage any requests that come in for new API domains. The fact we handle the technical delegation of said requests is a quirk from when CDDO was part of GDS, I think?

Anyhow, we're in conversation with CDDO about moving over api.gov.uk to them. We're expecting an update on that by the end of February.

Reply:
Ahh, that is a bit of a wat, but I think I was confused by "delegate it back":

```suggestion
The domain itself is owned by Cabinet Office, but delegated to GOV.UK, who then delegate subdomains (such as www.api.gov.uk) back to Cabinet Office, and other subdomains (such as driver-vehicle-licensing.api.gov.uk) to other organisations.
```